ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-ref.yaml

```diff
@@ -29,5 +29,14 @@ ref:
   - name: TEST_RELEASE_TYPE
     default: "GA"
     documentation: "The testing that is being carried out is for GA builds or Pre-GA builds"
+  - name: WORKLOAD_IMAGE
+    default: "quay.io/openshift/origin-hello-openshift"
+    documentation: "The workload image."
+  - name: WORKLOAD_TO_TEST
+    default: "kata"
+    documentation: |-
+      What kind of cluster is built. It still needs correct settings for RUNTIMECLASS and
```

Comment:
IIUC this option is only used to detect which tests should be executed? If so then I wouldn't start with the "cluster is built" as it might be understood as that this option has something to do with that. How about "What kind of workload you intend to be testing (kata, peer-pods, coco). Note: you still have to set RUNTIMECLASS and ENABLEPEERPODS variables.

Reply:
@ldoktor  The RUNTIMECLASS is used when the pod workloads are created.  It is possible to run kata pods when peer-pods are enabled.  If nested virt is on the cluster.

Before coco, we could key off ENABLEPEERPODS or RUNTIMECLASS to skip tests.  All of the peer-pod tests should run on a coco cluster.

But we need to install trustee on the cluster for coco.  There are some tests that only make sense for coco and not peer-pods.   And it needs a signed pod.  So we added WORKLOADTOTEST.